setup.py

```diff
@@ -25,7 +25,6 @@ def readme(file_name):
     py_modules=["nb_black", "lab_black"],
     zip_safe=False,
     install_requires=[
-        "yapf >= 0.28; python_version < '3.6'",
         "black >= 19.3; python_version >= '3.6'",
```

Comment:
From the error message in #40 , it seems like we might need to change the next line as well. 

Error message from #40 
```
Expected end or semicolon (after name and no valid version specifier)
```
```suggestion
        "black >= 19.3",
```